web/source/kmwosk.js

```diff
@@ -2801,6 +2800,11 @@
     
     for(nLayer=0;nLayer<layers.length; nLayer++)
     {
+      // Check the heights of each row, in case different layers have different row counts.
+      nRows=layers[nLayer].childNodes.length;
+      rowHeight=Math.floor(oskHeight/nRows);
```

Comment:
I doubt that Keyman Developer would let that happen... but a quick check on nRows couldn't hurt.  I did a quick inline `nRows == 0 ? 1 : nRows` in an updated commit.